synfig-studio/src/gui/app.cpp

```diff
@@ -1310,7 +1319,7 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	ACCEL("i",								"<Actions>/action_group_state_manager/state-mirror"			);
 	ACCEL("e",								"<Actions>/action_group_state_manager/state-circle"			);
 	ACCEL("r",								"<Actions>/action_group_state_manager/state-rectangle"			);
-	ACCEL("asterisk",								"<Actions>/action_group_state_manager/state-star"			);
```

Comment:
you didn't have to touch this part. Now FirasH and your PR are in conflict. Because both of you are modifying the same line.

Reply:
@Keyikedalube Ohh Sorry, Now what should i do ?? will undoing the change on this line solve the problem ?? I guess, it should !!

Reply:
Resolved :)